src/App.js

```diff
@@ -16,12 +19,16 @@ class App extends React.Component {
     }
   }
 
+
+
   render(){
     return (
       <div className="App">
         <Router>
           <Route path="/login" exact component={Login} />
           <Route path="/signup" exact component={Signup} />
+          <Route path="/bookclub/:id" exact component={BookClubs} />
```

Comment:
It would be best if you made these routes with a condition checking if the user is signed in and redirect them to sign in if they are not. For example:
```javascript
 <Route
              exact
              path="/bookclub/:id"
              render={() =>
                !sessionStorage.JWT ? <Redirect to="/login" /> : <MainPage />
              }
            />
```